package.json

```diff
@@ -12,6 +12,7 @@
     "/release"
   ],
   "main": "release/connect-streams.js",
+  "types": "src/index.d.ts",
```

Comment:
See [here](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package) for more info.